data/json/items/armor/ballistic_armor.json

```diff
@@ -39,7 +39,7 @@
     "coverage": 85,
     "encumbrance": 10,
     "warmth": 15,
-    "material_thickness": 20,
+    "material_thickness": 16,
```

Comment:
This is wrong. ESAPI plates are 20mm thick on the lowest end.

Reply:
Is that what material thickness is supposed to represent? I was under the impression it's a magic number for determining armor values. If so, ceramic or nylon need changes to prevent this from getting a buff if that is changed back.

Reply:
Just to outline how I see this working real quick, I'd like to standardize on a physical thickness measurement (ideally with units etc), and then we loop back and have some unit tests where we essentially recreate various ballistic acceptance testing procedures like, "this will block 3 rounds of 9mm Parabellum".